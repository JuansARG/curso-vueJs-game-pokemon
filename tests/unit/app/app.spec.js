import app from '@/App'
import { shallowMount } from '@vue/test-utils'


describe('test app.vue', () => {

    let wrapper

    beforeEach( () => {
        wrapper = shallowMount( app )
    })

    test('Debe hacer match con el snapshot', () => {

        expect( wrapper.html() ).toMatchSnapshot()

    })

})