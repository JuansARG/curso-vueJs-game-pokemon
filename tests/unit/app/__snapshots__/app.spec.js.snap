// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test app.vue Debe hacer match con el snapshot 1`] = `<pokemon-page-stub></pokemon-page-stub>`;
